Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,121 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //initialize my variables
+            string firstName = "";
+            string lastName = "";
+            int age = 0;
+            float weightInKg = 0f;
+            float height = 0f;
+
+            //ask for first input of name
+            // then send call method to make sure it fits name paramter of only letters and assign to name variable
+            Console.WriteLine("Please enter your first name.");
+            firstName = CheckStringForLettersOnly(firstName);
+
+            // same as first name but for last name
+            Console.WriteLine("Please enter your last name.");
+            lastName = CheckStringForLettersOnly(lastName);
+
+
+            // same as name inputs but for age
+            //call age method and check to make sure it fits parameter of numbers only.
+            Console.WriteLine("Please enter your age.");
+            age = int.Parse(CheckStringForDigitsOnly(age.ToString()));
+
+            //same as age
+            Console.WriteLine("Please enter your weight in kg.");
+            weightInKg = float.Parse(CheckStringForDigitsOnly(weightInKg.ToString()));
+
+            //same as age
+            Console.WriteLine("Please enter your height in cm.");
+            height = float.Parse(CheckStringForDigitsOnly(height.ToString()));
+
+
+            //after all values added run this at end of program
+            Console.WriteLine("Thank you for your information. You entered:");
+
+            //now dispaly the data that was given in a sentence. 
+            Console.WriteLine($"\n``` {firstName} {lastName} is {age} years old, your weight is {weightInKg} kg and" +
+                              $" your height is {height} cm. ```");
+
+            //tell user to press anybutton to close
+            Console.WriteLine("Press any button to exit.");
+
+            //wait for the button to be pressed
+            Console.ReadKey();
+        }
+
+        //Method that checks if string has only letters in it
+        private static string CheckStringForLettersOnly(string testString)
+        {
+            bool stop = true;
+
+            while (stop)
+            {
+                //read the input from console this is what will be tested for only letters
+                testString = Console.ReadLine();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Introduce a new variable instead of reusing the parameter 'testString'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4837469)